x-pack/plugins/apm/ftr_e2e/cypress/plugins/index.ts

```diff
@@ -42,12 +43,25 @@ const plugin: Cypress.PluginConfig = (on, config) => {
     client,
     createLogger(LogLevel.info),
     {
-      forceLegacyIndices: true,
+      forceLegacyIndices: false,
       refreshAfterIndex: true,
     }
   );
 
   on('task', {
+    // send logs to node process
+    log(message) {
```

Comment:
`log` and `wait` don't seem to be used.

Reply:
They are both utility functions that are useful for debugging. I think we can leave them in.

Reply:
I'm not in favour of having unused code but not in the fence for this one